aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -84,6 +86,9 @@ module aptos_framework::object {
     /// derivation to produce an object address.
     const OBJECT_FROM_SEED_ADDRESS_SCHEME: u8 = 0xFE;
 
+    /// Address where unwanted objects can be forcefully transferred to.
+    const BURN_ADDRESS: address = @0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff;
```

Comment:
I wonder how we choose between 0x0 and 0xffffffffff.... 0 is not that error prone to me.

Reply:
Also, does it matter if this is a reserved framework address or not?